src/core/index.js

```diff
@@ -62,6 +63,7 @@ class IPFS {
     this.bitswap = components.bitswap(this)
     this.ping = components.ping(this)
     this.pubsub = components.pubsub(this)
+    this.repoExists = utils.repoExists(this)
```

Comment:
Instead of adding a new utility function, we could just assign `this.repoExists` to `repoInstance.exists` and all should work. Would be less duplicated code + tests.